README-DVC.md

```diff
@@ -577,111 +596,6 @@ dvc run --file outputs.tcga.expression.dvc --yes \
   -o outputs/tcga/UVM.ExpressionOf.Edge.json.gz \
   "python3 transform/tcga/expression.py"
 #
-dvc run --file outputs.tcga.gistic2cna.dvc --yes \
```

Comment:
Why is this block being removed?

Reply:
I'll check. I generated the commands from the files...

Reply:
Found the issue on how I was regenerating the provenance commands.

Reply:
Added fix for #271